ppa/archive/tests/test_hathi.py

```diff
@@ -222,43 +222,41 @@ class TestHathiObject:
 
     ht_tempdir = tempfile.TemporaryDirectory(prefix="ht_text_pd")
 
-    def test_pairtree_prefix(self):
+    def test_lib_id(self):
+        hobj = hathi.HathiObject(hathi_id="uva.1234")
+        assert hobj.lib_id == "uva"
+
+    def test_vol_id(self):
         hobj = hathi.HathiObject(hathi_id="uva.1234")
-        assert hobj.pairtree_prefix == "uva"
+        assert hobj.vol_id == "1234"
 
-    def test_pairtree_id(self):
+    def test_pairtree_prefix(self):
         hobj = hathi.HathiObject(hathi_id="uva.1234")
-        assert hobj.pairtree_id == "1234"
+        assert hobj.pairtree_prefix == "uva."
```

Comment:
these could all be tested in a `test_init` method now that they aren't implemented as separate properties (but it's fine to leave as is!)

Reply:
I was debating this, I might as well make the change while I'm cleaning up this particular section.